lib/floating_animation/glare_effect_widget.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+import '../constants.dart';
+
+class GlareEffectWidget extends StatelessWidget {
+  const GlareEffectWidget({
+    required this.child,
+    this.glarePosition,
+    this.border,
+    super.key,
+  });
+
+  final Widget child;
+  final BoxBorder? border;
+  final double? glarePosition;
+
+  static final List<Color> _glareGradientColors = <Color>[
+    AppConstants.defaultGlareColor.withOpacity(0.1),
+    AppConstants.defaultGlareColor.withOpacity(0.07),
+    AppConstants.defaultGlareColor.withOpacity(0.05),
+  ];
```

Comment:
Instead of static member can we create a getter for this?

Reply:
Creating getter would cause creation of the object in question again and again on every usage of it. I believe using this way would save that case.